src/hb-ot-var-common.hh

```diff
@@ -1222,9 +1222,8 @@ struct TupleVariationData
       TRACE_SERIALIZE (this);
       for (const auto& tuple: tuple_vars)
       {
-        hb_bytes_t compiled_bytes {tuple.compiled_tuple_header.arrayZ, tuple.compiled_tuple_header.length};
-        compiled_bytes.copy (c);
-        if (c->in_error ()) return_trace (false);
+        if (!c->embed (tuple.compiled_tuple_header.arrayZ, tuple.compiled_tuple_header.length))
```

Comment:
Why not use `compiled_tuple_header.copy` here?

Reply:
oh I did not find a copy in hb_vector_t. If we use serializer's copy, it copies the length of the vector as well, and I'd like to avoid that

Reply:
Yeah I think if you do `as_array().copy()` that works.

Reply:
oh...I see! Then it's just a cosmetic change now. no functional change.